onnxruntime/core/session/onnxruntime_c_api.cc

```diff
@@ -3445,6 +3495,17 @@ ORT_API_STATUS_IMPL(OrtApis::GetEpDevices, _In_ const OrtEnv* /*env*/,
   API_IMPL_END
 }
 
+// Minimal build stub for GetEpCompatibilityForDevices to satisfy symbol references from the API table
+ORT_API_STATUS_IMPL(OrtApis::GetEpCompatibilityForDevices,
+                    _In_reads_(num_ep_devices) const OrtEpDevice* const* /*ep_devices*/,
+                    _In_ size_t /*num_ep_devices*/,
+                    _In_ const char* /*compatibility_info*/,
+                    _Out_ OrtCompiledModelCompatibility* /*out_status*/) {
+  API_IMPL_BEGIN
+  return OrtApis::CreateStatus(ORT_NOT_IMPLEMENTED, "This API in not supported in a minimal build.");
```

Comment:
nit: can include the function name in the error message: `"GetEpCompatibilityForDevices is not supported in a minimal build."`